packages/kbn-guided-onboarding/src/components/landing_page/guide_card_footer.test.tsx

```diff
@@ -23,7 +23,7 @@ const searchGuideState: GuideState = {
   status: 'not_started',
   steps: [
     { id: 'add_data', status: 'complete' },
-    { id: 'browse_docs', status: 'in_progress' },
+    { id: 'search_experience', status: 'in_progress' },
```

Comment:
These tests should probably be updated to use the test guide config, but I think it's out of scope for this PR.